api_types.ts

```diff
@@ -761,6 +761,12 @@ export interface SyncExecutionContext extends ExecutionContext {
   readonly sync: Sync;
 }
 
+export interface CellAutocompleteExecutionContext extends ExecutionContext {
+  readonly propertyName: string;
+  readonly propertyValues: {[propertyValues: string]: any};
+  readonly search: string;
```

Comment:
For parameter auto-complete the search value is passed as a parameter to the `autocomplete` function:

```ts
autocomplete: async function (context, search, parameters) {
  // ...
},
```

For consistency I'd recommend doing the same for cell auto-complete. I'm not opposed to also including it in the context, if that is what you are doing here.

Reply:
The reason for including it in the context here is to do the implicit caching trickery where we can see if the search string was used or not and factor that into the cache key.

With a parameter I'm not sure if that'd be possible, but with a property we can use `Object.defineProperty()` to spy on reads